pkg/rancher-components/src/components/Form/Checkbox/Checkbox.test.ts

```diff
@@ -0,0 +1,77 @@
+import { shallowMount } from '@vue/test-utils';
+import { Checkbox } from './index';
+
+describe('Checkbox.vue', () => {
+  it('renders falsy by default', () => {
+    const wrapper = shallowMount(Checkbox);
+    const cbInput = wrapper.find('input[type="checkbox"]').element as HTMLInputElement;
+
+    expect(cbInput.checked).toBeFalsy();
```

Comment:
Would `toBe(false)` make more sense here? I think `cbInput.checked` should be a `boolean` type.

Reply:
For the default value `toBe(false)` sounds right, fixed.